packages/wallets/src/components/WalletStatusHeader.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Trans } from '@lingui/macro';
+import { Flex, StateColor, StateIndicatorDot } from '@chia/core';
+import { useGetWalletConnectionsQuery } from '@chia/api-react';
+import { Box, ButtonGroup, Button } from '@mui/material';
+import WalletStatus from './WalletStatus';
+import { useTheme } from '@mui/styles';
+
+export default function WalletStatusHeader() {
+  const theme = useTheme();
+  const { data: connections } = useGetWalletConnectionsQuery({
+    walletId: 1,
```

Comment:
Is the walletId meant to be hard-coded here?

Reply:
I guess it doesn't really matter, if you're just trying to get the wallet connections.